src/core_plugins/kibana/public/discover/components/field_chooser/lib/detail_views/string.html

```diff
@@ -45,10 +45,11 @@ <h5 ng-show="!field.details.error">Quick Count <kbn-info info="Top 5 values base
       <a
         ng-href="{{field.details.visualizeUrl}}"
         ng-show="field.visualizable"
-        class="sidebar-item-button primary"
-        data-test-subj="fieldVisualize-{{::field.name}}">
-          Visualize
-          <span class="discover-field-vis-warning" ng-show="warnings.length" tooltip="{{warnings.join(' ')}}">
-          ( {{::warnings.length}} <ng-pluralize count="warnings.length" when="{'1':'warning', 'other':'warnings'}"></ng-pluralize> <i aria-hidden="true" class="fa fa-warning"></i> )
-          </span>
+        class="kuiButton kuiButton--primary kuiButton--small kuiButton--fullWidth"
+        data-test-subj="fieldVisualize-{{::field.name}}"
+      >
+        Visualize
```

Comment:
Ah, you got me! Great eye.

![image](https://cloud.githubusercontent.com/assets/1238659/25299650/3f844194-26b6-11e7-8421-13d5859c2a71.png)